contributions.md

```diff
@@ -15,22 +15,34 @@ We are looking for contributions in the following areas:
 
 ## Nuance
 
-This section outlines 2-4 common pitfalls, challenges, or limitations teams commonly encounter when adopting this practice. Each nuance should have a subheader and description.
+This next section outlines 2-4 common pitfalls, challenges, or limitations that teams commonly encounter when adopting this practice. Each nuance should have a title (styled as a subhead) and a brief (3-5 sentences) description.
 
-## How To Improve
+*Begin this section with the following text:* "This section outlines common pitfalls, challenges, or limitations teams commonly encounter when applying this practice. The goal here is not to discourage you. Rather, the goal is to arm you with the appropriate context so that you can make an informed decision about when and how to implement the practice with your team(s)."
 
-List practices from [Learning Culture](/capabilities/learning-culture.md) and add the practice specific information (talking points, instructions, links to external resources, etc) in the description. See the How To Improve sections in [other practice pages](/practices/) for examples.
+## Putting the Practice to Use [or "Practice in Action"?]
+
+*Begin this section with the following text:* "The following actions will help your team implement this practice." 
+
+Then, list the relevant general practices - from [Learning Culture](/capabilities/learning-culture.md) - and how to put those general practices to use by adding specifics such as talking points, demonstration instructions, roundtable discussion prompts, and links to external resources. 
+
+For example, a general practice is to Facilitate a Roundtable Discussion. In the Run Pair Programming Sessions practice, we elaborate on how to facilitate a roundtable discussion by listing specific discussion prompts related to pair programming such as, "How frequently do we engage in pair programming sessions, and are they integrated into our regular workflow?" 
+
+Another example of a general practice is to Start a Book Club. In the Reduce Coupling Between Abstractions practice, we list specific books that can help put this practice to use such as "Refactoring" by Martin Fowler and "Clean Architecture" by Robert C. Martin.
+
+See the relevant section in [other practice pages](/practices/) for more examples.
 
 ## Adjacent Capabilities
 
-2-4 other DORA Capabilities that are related either because they're similar or because you need one to accomplish the other. Each capability should have a title and description. The title should be an already existing DORA Capability.
+The final section lists 2-4 other DORA Capabilities that are related to the Practice, either because they're similar or because you need one to accomplish the other. Each Capability should have a title (styled as a subhead) and brief (2-4 sentences) description. The title should be an  existing, linked DORA Capability from the repository.
```

Comment:
Some practices (run pair programming and reduce coupling) have only ONE adjacent capability. Should those be revisited, or should this description be more flexible to allow for just one? 
e.g., ("The final section lists 1-4 other DORA Capabilities that are related to the Practice...")

Reply:
we can update to only have 1 in the case where that makes sense